### DISCUSSION TITLE
higress 是否支持哪几种协议

### DISCUSSION THREAD

**Discussion by Notailli** [alibaba/higress#356] [Q&A]
HTTP、HTTPS、TCP、UDP、Dubbo、gRPC，哪些支持

**Comment by johnlanni** [comment]
UDP不支持，Dubbo 只支持 HTTP to Dubbo，其他都可以

**Reply by zhuchance** [reply]
Does it support Redis  RESP protocol？

**Reply by johnlanni** [reply]
Did you mean HTTP to RESP? Or does it refer to the Redis Proxy scenario?

**Reply by zooyf** [reply]
现在支持了吗？如果没有，有计划什么时候支持吗？

---
Is it supported now? If not, will there be plans to support it?

**Reply by johnlanni** [reply]
可以用gateway api的udp proxy CRD。不过注意 gateway api 的版本目前还只支持 1.0.0，不能安装最新的
